Adyen/client.py

```diff
@@ -87,8 +87,8 @@ def __init__(self, username=None, password=None, xapikey=None,
         self.skin_code = skin_code
         self.psp_list = []
         self.app_name = app_name
-        self.LIB_VERSION = "1.4.0"
-        self.USER_AGENT_SUFFIX = "adyen-python-api-library/"
+        self.LIB_VERSION = settings.LIB_VERSION
```

Comment:
do we need to assign settings.LIB_VERSION ? do we need self.LIB_VERSION ?

Reply:
self.LIB_VERSION is the one that belong to the class. The settings.LIB_VERSION it is the one that we set as constant.

Reply:
yes it is needed. It is used by the HTTPClientto create the full 
 self.user_agent = app_name + " " + user_agent_suffix + lib_version